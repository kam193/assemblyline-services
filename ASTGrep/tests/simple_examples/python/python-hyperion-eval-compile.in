eval(compile(str("\x67\x6c\x6f\x62\x61\x6c\x73\x28\x29\x5b\x27\x5c\x78\x36\x35\x5c\x78\x37\x36\x5c\x78\x36\x31\x5c\x78\x36\x63\x27\x5d(NMNNMMMNNMNNMNMMNMNMNNNM)"),filename='\x57\x57\x57\x58\x57\x57\x57\x58\x58\x57\x57\x57\x58\x57\x57\x58\x58\x58\x58',mode='\x65\x76\x61\x6c'))

WWXXWWXXWXWWXXXWXXWW,S222222SSSS22SSSS22222S222,nnmnnmnnmmmnnmnmnnnmmmnn,OoO00OOoo000Oo0OOo0,DOoOoODoOooOoOooo=(lambda NMNNMMMNNMNNMNMMNMNMNNNM:NMNNMMMNNMNNMNMMNMNMNNNM(zlib)),(lambda NMNNMMMNNMNNMNMMNMNMNNNM:eval(compile(str("\x67\x6c\x6f\x62\x61\x6c\x73\x28\x29\x5b\x27\x5c\x78\x36\x35\x5c\x78\x37\x36\x5c\x78\x36\x31\x5c\x78\x36\x63\x27\x5d(NMNNMMMNNMNNMNMMNMNMNNNM)"),filename='\x57\x57\x57\x58\x57\x57\x57\x58\x58\x57\x57\x57\x58\x57\x57\x58\x58\x58\x58',mode='\x65\x76\x61\x6c'))),(lambda NMNNMMMNNMNNMNMMNMNMNNNM:NMNNMMMNNMNNMNMMNMNMNNNM['\x64\x65\x63\x6f\x6d\x70\x72\x65\x73\x73']),(lambda:(lambda NMNNMMMNNMNNMNMMNMNMNNNM:exec(compile(str("\x67\x6c\x6f\x62\x61\x6c\x73\x28\x29\x5b\x27\x5c\x78\x36\x35\x5c\x78\x37\x36\x5c\x78\x27\x5d(NMNNMMMNNMNNMNMMNMNMNNNM)"),filename='\x57\x57\x57\x58\x57\x57\x57\x58\x58\x57\x57\x57\x58\x57\x57\x58\x58\x58\x58',mode='\x65\x76\x61\x6c')))('\x5f\x5f\x69\x6d\x70\x6f\x72\x74\x5f\x5f\x28\x27\x62\x75\x69\x6c\x74\x69\x6e\x73\x27\x29\x2e\x65\x78\x65\x63')),(lambda O00oO00oOOO00O00o0O0o0Ooo,NMNNMMMNNMNNMNMMNMNMNNNM:O00oO00oOOO00O00o0O0o0Ooo(NMNNMMMNNMNNMNMMNMNMNNNM))

eval(compile("\x67\x6c\x6f\x62\x61\x6c\x73\x28\x29\x5b\x27\x5c\x78\x36\x35\x5c\x78\x37\x36\x5c\x78\x36\x31\x5c\x78\x36\x63\x27\x5d(NMNNMMMNNMNNMNMMNMNMNNNM)",filename='\x57\x57\x57\x58\x57\x57\x57\x58\x58\x57\x57\x57\x58\x57\x57\x58\x58\x58\x58',mode='\x65\x76\x61\x6c'))

eval(compile(something('\x74\x72\x75\x73\x74')))

exec(compile("def _raise(*a): raise a[0], a[1], a[2]", "<py3fix>", "exec"))
