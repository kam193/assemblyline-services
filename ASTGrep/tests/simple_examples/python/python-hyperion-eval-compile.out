eval(NMNNMMMNNMNNMNMMNMNMNNNM)

WWXXWWXXWXWWXXXWXXWW,S222222SSSS22SSSS22222S222,nnmnnmnnmmmnnmnmnnnmmmnn,OoO00OOoo000Oo0OOo0,DOoOoODoOooOoOooo=(lambda NMNNMMMNNMNNMNMMNMNMNNNM:NMNNMMMNNMNNMNMMNMNMNNNM(zlib)),(eval),(lambda NMNNMMMNNMNNMNMMNMNMNNNM:NMNNMMMNNMNNMNMMNMNMNNNM['\x64\x65\x63\x6f\x6d\x70\x72\x65\x73\x73']),(lambda:globals()['\x65\x76\x']('\x5f\x5f\x69\x6d\x70\x6f\x72\x74\x5f\x5f\x28\x27\x62\x75\x69\x6c\x74\x69\x6e\x73\x27\x29\x2e\x65\x78\x65\x63')),(lambda O00oO00oOOO00O00o0O0o0Ooo,NMNNMMMNNMNNMNMMNMNMNNNM:O00oO00oOOO00O00o0O0o0Ooo(NMNNMMMNNMNNMNMMNMNMNNNM))

eval(NMNNMMMNNMNNMNMMNMNMNNNM)

eval(compile(something('\x74\x72\x75\x73\x74')))
